index.html

```diff
@@ -3,14 +3,105 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>Great Idea - Responsive I</title>
+
+  <title>Great Idea!</title>
 
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
 </head>
 
-<body>
-<!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
-</body>
+  <body>
+
+    <!-- Copy and paste your HTML from the first UI project here -->
+
+    <div class="container">
+      
+      <header>
+        <nav>
+          <a href="services.html">Services</a>
+          <a href="#">Product</a>
+          <a href="#">Vision</a>
+          <a href="#">Features</a>
+          <a href="#">About</a>
+          <a href="#">Contact</a>
+          <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+        </nav>  
+      </header>
+
+    <div class="parent1">
+
+      <div class="child1">
+        <h1 class="index">Innovation<br>
+        On<br>
+        Demand</h1>
+        <button class="b1">Get Started</button>
+      </div>
+      <div class="child5">
+        <img src="img/header-img.png" alt="Image of a code snippet.">
```

Comment:
You can make this image responsive using CSS by setting the max width to 100%, "display: block", and "height: auto" to keep the original ratio of the image.